trailblazer/mip/start.py

```diff
@@ -8,60 +8,58 @@
 LOG = logging.getLogger(__name__)
 
 CLI_OPTIONS = {
-    'family': {'option': '--family_id'},
     'config': {'option': '--config_file'},
     'priority': {'option': '--slurm_quality_of_service'},
     'email': {'option': '--email'},
     'base': {'option': '--cluster_constant_path'},
-    'dryrun': {
-        'option': '--dry_run_all',
-        'default': '2',
-    },
-    'gene_list': {'option': '--vcfparser_select_file'},
-    'max_gaussian': {
-        'option': '--gatk_variantrecalibration_snv_max_gaussians',
-        'default': '1',
-    },
+    'dryrun': {'option': '--dry_run_all'},
+    'gene_list': {'option': '--vcfparser_slt_fl'},
+    'max_gaussian': { 'option': '--gatk_varrecal_snv_max_gau'},
     'skip_evaluation': {'option': '--qccollect_skip_evaluation'},
-    'start_with': {'option': '--start_with_program'},
+    'start_with': {'option': '--start_with_recipe'},
 }
 
 
 class MipCli(object):
 
     """Wrapper around MIP command line interface."""
 
-    def __init__(self, script):
+    def __init__(self, script, pipeline):
         """Initialize MIP command line interface."""
         self.script = script
+        self.pipeline = pipeline
 
-    def __call__(self, config, family, **kwargs):
+    def __call__(self, config, case, **kwargs):
         """Execute the pipeline."""
-        command = self.build_command(config, family=family, **kwargs)
+        command = self.build_command(case, config, **kwargs)
         LOG.debug(' '.join(command))
         process = self.execute(command)
         process.wait()
         if process.returncode != 0:
             raise MipStartError('error starting analysis, check the output')
         return process
 
-    def build_command(self, config, **kwargs):
+    def build_command(self, case, config, **kwargs):
         """Builds the command to execute MIP."""
-        command = ['perl', self.script, CLI_OPTIONS['config']['option'], config]
+        command = [self.script, self.pipeline, case, CLI_OPTIONS['config']['option'], config]
+        print(command)
```

Comment:
Is this left over form debugging? `print(command)`?

Reply:
Yes. refresh the page, you're looking at an old commit ;)